pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -15,10 +15,10 @@ import (
 )
 
 const (
-	defaultAPIVersion     = "cloudwatch.aws.amazon.com/v1alpha1"
-	defaultInstrumenation = "java-instrumentation"
-	defaultNamespace      = "default"
-	defaultKind           = "Instrumentation"
+	defaultAPIVersion      = "cloudwatch.aws.amazon.com/v1alpha1"
+	defaultInstrumentation = "java-instrumentation"
```

Comment:
we..should change the name of the instrumentation to `application-signals-instrumentation` or something. non blocking tho